test/lit.cfg

```diff
@@ -888,6 +895,12 @@ elif run_os in ['linux-gnu', 'linux-gnueabihf', 'freebsd', 'windows-cygnus', 'wi
       config.target_shared_library_prefix = 'lib'
       config.target_shared_library_suffix = ".so"
       config.target_sdk_name = "freebsd"
+    elif run_android:
```

Comment:
```
elif kIsAndroid:
```

Reply:
Done, see latest commit. This pull was tested against the latest 3/10 Swift 5.0 snapshot, cherry-picked to master and then I changed the variable name, so I missed this. However, it was tested with the old variable name, so it should work fine.